### DISCUSSION TITLE
Non-Local Images

### DISCUSSION THREAD

**Discussion by jshall77** [aaronwmorris/indi-allsky#1306] [Q&A]
I would like to use my FTP/SFTP/SSH server to serve images from.  I see that there is a way to do this for other storage(S3, etc) but it doesn't seem to work for local network options.  Is this possible or can it be added in a future release?  Thanks!

**Comment by aaronwmorris** [comment]
With the current file transfer functionality, remote images will not work.  The file transfer functionality was designed well before indi-allsky had a fully functional web interface as it has today.  The file transfer functionality was actually designed to integrate with TJ's Allsky remote web site.

I would have to add a different file transfer mode that uploaded all files with specific name on fixed paths so that the remote file paths would match a remote web server.  All remote files would ultimately have to be served via HTTP/HTTPS.
